mkdocs.yml

```diff
@@ -0,0 +1,87 @@
+site_name: Strawberry Django +
+site_description: Strawberry-Django with extra features
+#site_url: https://nrbnlulu.github.io/strawberry-django-auth/
```

Comment:
What will be the final site_url?

Reply:
See above:
>  Some configurations still will be needed to be done in the main repo.
but it should be somthing like blbventures.github.io/strawberry-django-plus